### PR TITLE
Penguins are real.

### PR DESCRIPTION
**Pull request by erichurkman**
🐧

### CODE REVIEW

**Review by gregorygmwhite — APPROVED** [review]
:ship: all the penguins

**Review by erickmendonca — APPROVED** [review]
Looks legit

File: test/test_cumberbatch.py [hunk 1]
```diff
@@ -86,3 +88,8 @@ def test_firstname_startswith_b_or_lastname_startswith_c(self):
                 self.assertTrue(full_first.startswith('B') or full_last.startswith('C'))
             except ValueError, e:
                 raise ValueError(fullname)
+
+    def test_penguins_are_real(self):
+        for i in range(1000):
+            penguin = cumberbatch.penguin()
+            self.assertTrue(penguin.startswith('Peng'))
```
Comment (mannysz) [inline test/test_cumberbatch.py:95]: The best unit test I've seen recently! 🐧

**Comment by brandones** [comment]
Unfortunately, we don't presently have the resources to support this functionality. QA is swamped as it is, and I'm pretty concerned with the maintenance overhead of our ever-expanding codebase. Perhaps once we finish a Series A we'll be able to hire a few Windows sysadmins and 12-18 Java-writing college freshmen interns, at which point I think we'll be in a good position to expand in new and promising directions like this one.

**Comment by erichurkman** [comment]
:(

**Pull request closed without merging**